src/Carousel/CarouselIndicators.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { act } from "react-dom/test-utils";
+import { unmountComponentAtNode, render } from "react-dom";
+import { CarouselIndicators } from "./CarouselIndicators";
+import { randomId } from "@sebgroup/frontend-tools/dist/randomId";
+
+describe("Component: Carousel", () => {
+    let container: HTMLDivElement = null;
+
+    beforeEach(() => {
+        container = document.createElement("div");
+        document.body.appendChild(container);
+    });
+
+    afterEach(() => {
+        unmountComponentAtNode(container);
+        container.remove();
+        container = null;
+    });
+
+    it("Should render", () => {
+        act(() => {
+            render(<CarouselIndicators />, container);
+        });
+        expect(container.firstElementChild).toBeDefined();
```

Comment:
if it's null, the test will still be successful
```suggestion
        expect(container.firstElementChild).not.toBeNull();
```